crates/rattler_conda_types/src/match_spec/parse.rs

```diff
@@ -820,4 +855,53 @@ mod tests {
         let spec = MatchSpec::from_str("conda-forge::foo", Strict).unwrap();
         assert!(spec.namespace.is_none());
     }
+
+    #[test]
+    fn test_parsing_url() {
+        let spec = MatchSpec::from_str(
+            "https://conda.anaconda.org/conda-forge/linux-64/py-rattler-0.6.1-py39h8169da8_0.conda",
+            Strict,
+        )
+        .unwrap();
+
+        assert_eq!(spec.url, Some(Url::parse("https://conda.anaconda.org/conda-forge/linux-64/py-rattler-0.6.1-py39h8169da8_0.conda").unwrap()));
+        assert_eq!(
+            spec.file_name,
+            Some("py-rattler-0.6.1-py39h8169da8_0.conda".parse().unwrap())
+        );
+    }
+
+    #[test]
+    fn test_parsing_path() {
+        let spec = MatchSpec::from_str(
+            "C:/Users/user/conda-bld/linux-64/foo-1.0-py27_0.tar.bz2",
+            Strict,
+        )
+        .unwrap();
+        assert_eq!(
+            spec.url,
+            Some(
+                Url::parse("file:C:/Users/user/conda-bld/linux-64/foo-1.0-py27_0.tar.bz2").unwrap()
+            )
+        );
```

Comment:
```suggestion
        let spec = MatchSpec::from_str(
            "C:\\Users\\user\\conda-bld\\linux-64\\foo-1.0-py27_0.tar.bz2",
            Strict,
        )
        .unwrap();
        assert_eq!(
            spec.url,
            Some(
                Url::parse("file://C:/Users/user/conda-bld/linux-64/foo-1.0-py27_0.tar.bz2").unwrap()
            )
        );
```

Windows paths are backslashes. I added that here. :)